### PR TITLE
Fix homepage

### PR DESCRIPTION
**Pull request by austinmargarone**

### CODE REVIEW

**Review by brandonetter — CHANGES_REQUESTED** [review]
Looks much better on the services section, I would suggest changing the breakpoint to be a little later though, as it still becomes very narrow all the way until 1700px wide. Good decision to make it change shape at smaller screens- but it changes way before we even hit the figma design width.

image

"Portfolio" should be renamed to "Case studies" I think, and the width and spacing is questionable there, as well. I think you should change the breakpoint to before 1700px wide as there is a ton of empty space.

image

Your cards on the homepage no longer have the image cut-off on the right, which is fine. But they also no longer have centered text vertically. The image and text to the left are vertically misaligned:

image

Other than that this is coming along great. I'll send you the sanity cli link

File: components/homepage/Hero.tsx [hunk 1]
```diff
@@ -22,24 +22,15 @@ const Hero = (props: Props) => {
...
+            <button className="herobtn md:herobtnlg mb-[.88rem] flex rounded-[379.77006rem] bg-blue dark:bg-blue1 md:mb-0">
+              <p className="herotext text-white md:text-[1.125rem]">My Work</p>
+            </button>
+          </Link>
+          <Email />
         </div>
       </div>
-      <div className="heroimg md:heroimglg md:flex-row">
+      <div className="heroimg md:heroimglg md:w-1/2 md:flex-row">
         <Image src="/Hero.svg" width={757} height={541} alt="Hero computers" />
```
Comment (brandonetter) [inline components/homepage/Hero.tsx]:
```suggestion
        <Image src="/Hero.svg" width={757} height={541} alt="Hero computers" priority/>
```
This hero image is the main image you want to load as the website renders. So label it as priority to optimize page load.

https://nextjs.org/docs/pages/building-your-application/optimizing/images

image

**Review by brandonetter — CHANGES_REQUESTED** [review]

**Comment by austinmargarone** [comment]
@brandonetter For confirmation you're referring to the mobile size switching to medium sooner? How do i go about this? I've been trying to implement this but unsuccessful.

**Comment by austinmargarone** [comment]
@brandonetter Does this look better?

**Comment by brandonetter** [comment]
@austinmargarone Case studies looks perfect. For the services it looks much better at 1460 - 1700, but below that you can keep what you were doing to make it two across if you'd like. To be clear I wasn't saying your method of grouping was wrong- just that you should do it at a different width.

## Next:
image

The text is now lined up vertically with the image. Good work. But it looks a little strange with the lack of padding to the right of the image if there's no overflow. It's also not really proportioned correctly- there's way more vertical empty space in yours than the design. Look at them side by side:

image

**Comment by brandonetter** [comment]
Here's with a couple changes and using the overflow again:

image


Same changes without using the overflow if you removed it because you didn't like it:

image


I like the color choice of the golden/yellow with the deep blue. Looks great!

**Comment by brandonetter** [comment]
Getting even narrower also looks pretty good in my opinion. Preferably you just follow the design given- but if you want to deviate a ton like this please try to center items and keep things aligned.

image

**Comment by austinmargarone** [comment]
@brandonetter How's this look?

**Comment by brandonetter** [comment]
I like it- especially the final mobile version looks clean:
image


Good work.

**Pull request merged**